libgnucash/app-utils/options.scm

```diff
@@ -1688,32 +1688,46 @@
           (let ((option-hash (hash-ref section-hash name)))
             (if option-hash
                 option-hash
-                ; Option name was not found. Perhaps it was renamed ?
-                ; Let's try to map it to a known new name
+                                        ; Option name was not found. Perhaps it was renamed ?
+                                        ; Let's try to map it to a known new name.
+                                        ; This list will try match names - if one is found
+                                        ; the next item will describe a pair.
+                                        ; (cons newsection newname)
+                                        ; If newsection is #f then reuse previous section name.
```

Comment:
Is there a good reason for the huge indent here?

Reply:
emacs made me do it...